README.md

```diff
@@ -50,17 +50,44 @@ See [API.md](API.md) for the public API.
 
 [![Free egghead GROQ introduction course by John Lindquist](https://user-images.githubusercontent.com/6188161/142889665-fc04ac47-d0fa-492b-897b-4203c97e94ec.png)](https://egghead.io/courses/introduction-to-groq-query-language-6e9c6fc0?utm_source=github&utm_medium=cta&utm_term=GROQ)
 
-
 ## Versioning
 
-GROQ-JS follows [SemVer](https://semver.org) and is currently at version v0.2.
+GROQ-JS follows [SemVer](https://semver.org).
 See [the changelog](CHANGELOG.md) for recent changes.
 This is an "experimental" release and anything _may_ change at any time, but we're trying to keep changes as minimal as possible:
 
-- The public API of the parser/evaluator will most likely stay the same in future version.
+- The public API of the parser/evaluator will most likely stay the same in future versions.
 - The syntax tree is _not_ considered a public API and may change at any time.
 - This package always implements the latest version of [GROQ according to the specification](https://github.com/sanity-io/groq).
-- The goal is to release a v1.0.0 by the end of 2020.
+
+## Releasing a new version
+
+The following steps are required to release a new version. You should only release from the `main` branch.
+
+1. Update the CHANGELOG inline with the existing format (remember [SemVer](https://semver.org)!)
+2. Update the version field in `package.json` and commit it to version control
+3. Create a Git tag with the version number (`git tag vX.Y.Z`)
+4. Push upstream to the `main` branch of the GitHub repo
+5. Run `npm publish`
+6. Check the package was successfully published
```

Comment:
This can be shortened a bit with `np`

1. Update the CHANGELOG inline with the existing format (remember [SemVer](https://semver.org)!)
2. Run `npx np`
  	`np` will run the tests,  update package.json, create a git tag, push it to github and publish the package on npm

Reply:
But if you use `np` you shouldn't create a git tag since `np` will do it, same goes for updating package.json.

We can rather document what `np` is doing?

Reply:
If someone wants to understand the explicit steps to how we release `groq-js` they won't be able to figure that out by checking the README, they'll need to go read what `np` does.

That seems like uneccessary redirection when we could lay out the required steps, and then suggest they use the tool to make life easier.

I'll reword this to point readers to `np`, but with a brief explanation of the underlying steps.

Reply:
> We can rather document what np is doing?

Ahhhh that's what you suggested anyways!